src/features/questions/questions-list/FilteredQuestionGroups.tsx

```diff
@@ -21,6 +22,41 @@ const FadeInQuestionGroup = withFadeIn<QuestionGroupItemProps>(
 export const FilteredQuestionGroups: React.FC<{
   filterQuestionGroups: ReadonlyArray<QuestionGroup>
 }> = ({ filterQuestionGroups }) => {
+  const [focusedItem, setFocusedItem] = useState(1)
+  const [focusedInsideItem, setFocusedInsideItem] = useState(0)
+
+  const isUpPressed = useKeyIsPressed(Keys.Up)
+  const isDownPressed = useKeyIsPressed(Keys.Down)
+  const isEnterPressed = useKeyIsPressed(Keys.Enter)
+  const isOptionPressed = useKeyIsPressed(Keys.Option)
```

Comment:
As I mentioned in slack and in IP-67 the "form submission hotkey" should be `Command + Enter` and not `Option + Enter` (sorry for the mixed signals 🙈 )
```suggestion
  const isCommandPressed = useKeyIsPressed(Keys.Option)
```